src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -95,14 +117,55 @@ module.exports.commands = [{
         this.expect.element('@code').to.not.be.present.before(1000);
 
     },
+    validateGraph: function () {
+        var self = this;
+        self.waitForElementVisible('@highlightedGraph');
+        return self;
+    },
+    validateSteps: function (browser) {
+        var self = this;
+        self.waitForElementVisible('@logConsole');
+        browser.elements('css selector', '.logConsole', function (codeCollection) {
```

Comment:
I still don't get why we need to pas around the `browser` ... it's a while since I did this stuff but I could swear that was not needed. The fact that you can do `self.waitForElementVisible`/`this.waitForElementVisible` also suggests to me that it's not needed.